draft-ietf-quic-multipath.md

```diff
@@ -979,7 +979,9 @@ faster if the scheduling strategy is stable, but besides that
 implementations can choose between multiple strategies such as sending
 PATH_ACK frames on the path they acknowledge packets, or sending
 PATH_ACK frames on the shortest path, which results in shorter control loops
-and thus better performance.
+and thus better performance. However, since packets that only carries PATH_ACK frames
+are not congestion controlled, senders should carefully consider the load induced
```

Comment:
```suggestion
are not congestion controlled(following {{Section 7 of QUIC-RECOVERY}}), senders should carefully consider the load induced
```

Reply:
I think it's contained in RFC 9002 Section 7:
> Similar to TCP, packets containing only ACK frames do not count toward bytes in flight and are not congestion controlled.

But as PATH_ACK is a new frame added by multipath extension, I'm not against on adding this part to make things more clear. Just minor fix.

Reply:
I believe that the way RFC9002 is written is wrong. It wants to say that ACK can be sent whatever the state of congestion control, but it says that by stating that "ACK packets do count as bytes in flight", which is wrong -- they create queues in the network just like other packets. Mirja's text nicely avoids that issue. I do not think that we should add a reference to RFC9002.

Reply:
@huitema You're right about the packets only contains ACK still create queues on switching devices and cause process cost on both sides. My suggestion is about that "PATH_ACK is not count into congestion control" isn't something new added by multipath extension, but is inherit from RFC 9000 and 9002.

Reply:
Sec 7 of RFC9002 says:

"Similar to TCP, packets containing only ACK frames do not count toward bytes in flight and are not congestion controlled."

Further the next sentence in RFC9002 is actually:

"Unlike TCP, QUIC can detect the loss of these packets and MAY use that information to adjust the congestion controller or the rate of ACK-only packets being sent, but this document does not describe a mechanism for doing so."

So I guess it's fine to add a reference here or it might actually be useful for more context.

Reply:
Not really a comment on this PR, but the part about QUIC using acknowledgments of ACK packets may be a bit optimistic. ACK-only packets do not trigger ACK. Implementation may trigger that ACK by bundling a Ping with their ACK, but they cannot do that too frequently -- typically no more than once every N ACK, with N at least 2, and no more than once per RTT. So you get very scarce feedback, and any estimation of capacity based on that is imprecise. Plus, in the general case, the ACK traffic is way lower than link capacity, so you don't get an accurate estimate of capacity -- this is an app-limited scenario.

Reply:
Yes, ACK congestion control is an open issue but that's not specific for the multipath case, as also initially discussed in issue #454. However, given we discuss in this text in the implementation guidance explicitly that ACKs could be send on another path, with is different than in the single path case, I think it is fair to add an additional warning about congestion control and point to the respective text in RFC9002 that already takes about ACK congestion control. Therefore I propose to merge this PR incl. the proposed commit with the reference to RFC9002; I believe the discussed showed there is agreement to merge.